### DISCUSSION TITLE
How to get data returned by selecting a specific tab

### DISCUSSION THREAD

**Discussion by Muhammad-fareed** [filamentphp/filament#10446] [Help]
Hello i am stuck on a problem. I am new to filament i am trying to get the data returned by tabs i have created 2 tabs like this :
`public function getTabs(): array
    {

        return [
            'all' => Tab::make(),
            'This Week' => Tab::make()
                ->modifyQueryUsing(fn (Builder $query) => $query->where('created_at', '>=', now()->subWeek()))
                ->badge(Transaction::query()->where('created_at', '>=', now()->subWeek())->count()),
            'This Month' => Tab::make()
                ->modifyQueryUsing(fn (Builder $query) => $query->where('created_at', '>=', now()->subMonth()))
                ->badge(Transaction::query()->where('created_at', '>=', now()->subMonth())->count()),
        ];
    }`
i want to get the data returned by selecting a specific tab.

**Comment by leandrocfe** [comment]
Sorry, I didn't understand what you want. Could you provide more info?

**Reply by Muhammad-fareed** [reply]
i want to print a pdf with filtered data. But i am unable to get the data return by filters. Is there a method we can get access to filters.
